.gitignore

```diff
@@ -0,0 +1,13 @@
+*.xcuserstate
+**/xcuserdata/
+
+PatchDay.xcodeproj/project.xcworkspace/xcuserdata
```

Comment:
Please remove this as it is not relevant to this project. I think `**/xcuserdata/` will take care of this case.